ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/submission/CommandSubmitter.scala

```diff
@@ -163,9 +208,15 @@ case class CommandSubmitter(services: LedgerApiServices) {
 object CommandSubmitter {
   case class CommandSubmitterError(msg: String) extends RuntimeException(msg)
 
-  class ProgressMeter(totalItems: Int, startTimeMillis: Long) {
+  class ProgressMeter(totalItems: Int) {
+    var startTimeMillis: Long = System.currentTimeMillis()
+
+    def start(): Unit = {
```

Comment:
Note: explicit start method to move the start of measurement closer to the start of contract submission. There is still a ~0.2sec delay between the start call and the first submission, which leads to overestimated remaining times for the first few progress reports.